src/CommunityToolkit/Xamarin.CommunityToolkit.UnitTests/Helpers/LocalizedStringTests/LocalizedStringTests.cs

```diff
@@ -0,0 +1,70 @@
+﻿using System;
+using System.Globalization;
+using System.Resources;
+using Xamarin.CommunityToolkit.Helpers;
+using Xamarin.CommunityToolkit.UnitTests.Mocks;
+using Xunit;
+
+namespace Xamarin.CommunityToolkit.UnitTests.Helpers.LocalizedStringTests
+{
+	public class LocalizedStringTests
+	{
+		public LocalizedStringTests()
+		{
+			resourceManager = new MockResourceManager();
+			localizationManager = new LocalizationResourceManager();
```

Comment:
We shouldn't create new instances of this class. I already created a PR to fix it. And I just saw it thanks to your PR ❤
```suggestion
			localizationManager = LocalizationResourceManager.Current;
```

Reply:
Even for testing? A bit strange to call `Init()`on the same instance multiple times, but okay.